### PR TITLE
Remove deprecated import convention "Krakow_Hielscher" in from_euler()

### PR DESCRIPTION
**Pull request by pc494**
#### Description of the change
Removing a deprecation in advance of 0.8, see #230. Will let the CI find the tests that need removing.

#### Progress of the PR
- [x] [Docstrings for all functions](https://github.com/numpy/numpy/blob/master/doc/example.py)
- [x] Unit tests with pytest for all lines
- [x] Clean code style by [running black via pre-commit](https://orix.readthedocs.io/en/latest/contributing.html#code-style)

#### For reviewers

- [x] The PR title is short, concise, and will make sense 1 year later.
- [n/a] New functions are imported in corresponding `__init__.py`.
- [x] New features, API changes, and deprecations are mentioned in the
      unreleased section in `CHANGELOG.rst`.

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -61,6 +61,7 @@ Removed
 -------
 - `StereographicPlot` methods `azimuth_grid()` and `polar_grid()`.
   Use `stereographic_grid()` instead.
+- `fromo_euler()' no longer accepts "Krakow_Hielscher" as a convention, use "MTEX" instead.
```
Comment (hakonanes) [inline CHANGELOG.rst:64]:
```suggestion
- `from_euler()` no longer accepts "Krakow_Hielscher" as a convention, use "MTEX" instead.
```

**Review by hakonanes — CHANGES_REQUESTED** [review]
Will merge when the suggested change and a black run is done!

**Comment by hakonanes** [comment]
Will merge after checks pass.

**Review by hakonanes — APPROVED** [review]

**Comment by hakonanes** [comment]
Will also delete the branch after it's merged.

**Comment by pc494** [comment]
This was very lazy of me, thank you for fixing @hakonanes

**Comment by hakonanes** [comment]
No problem!

**Pull request merged**